ui/src/components/Dashboard/index.jsx

```diff
@@ -243,10 +243,10 @@ export default class Dashboard extends Component {
           </Typography>
         </div>
         <Divider />
-        <UserMenu />
+        {docs ? <UserMenu /> : <UserMenu navOpen />}
```

Comment:
Is `navOpen` being used?

Reply:
yes it is being used here you can refer previous commit  https://github.com/taskcluster/taskcluster/pull/280/commits/2f8194b1e4eb994e9bd613096cb77d997b682414

Reply:
I still don't see how it's being used. What is it doing? Didn't we remove it in https://github.com/taskcluster/taskcluster/pull/280#discussion_r261261456?

Reply:
Replace this line with `<UserMenu />`.